node/service/src/lib.rs

```diff
@@ -71,10 +60,33 @@ pub type ChainSpec = service::ChainSpec<GenesisConfig>;
 pub type ComponentClient<C> = Client<<C as Components>::Backend, <C as Components>::Executor, Block>;
 pub type NetworkService = network::Service<Block, <Factory as service::ServiceFactory>::NetworkProtocol, Hash>;
 
+/// A verifier that doesn't actually do any checks
+pub struct NoneVerifier;
```

Comment:
Using `()` for "null" implementations of traits by convention.

Reply:
Can't be done here: `impl<B: BlockT> Verifier<B> for ()` can only be done within the crate that defines `Verifier`. However we don't want to provide such a bad implementation in `substrate-network`.

Leaving it as is for now, as this is only temporarily to make the compiler happy and will be replaced in the next chunk of work with an actual `Verifier` implementation.